com.unity.formats.fbx/Editor/FbxExporter.cs

```diff
@@ -1023,19 +1023,46 @@ SkinnedMeshRenderer unitySkin
         /// <returns>The bind pose.</returns>
         /// <param name="unityBone">Unity bone.</param>
         /// <param name="bindPoses">Bind poses.</param>
-        /// <param name="boneDict">Dictionary of bone to index.</param>
-        /// <param name="skinnedMesh">Skinned mesh.</param>
+        /// <param name="boneInfo">Contains information about bones and skinned mesh.</param>
         private Matrix4x4 GetBindPose(
             Transform unityBone, Matrix4x4[] bindPoses,
-            Dictionary<Transform, int> boneDict, SkinnedMeshRenderer skinnedMesh
-        ){
+            ref SkinnedMeshBoneInfo boneInfo
+        )
+        {
+            var boneDict = boneInfo.boneDict;
+            var skinnedMesh = boneInfo.skinnedMesh;
+            var boneToBindPose = boneInfo.boneToBindPose;
+
             Matrix4x4 bindPose;
+            if(boneToBindPose.TryGetValue(unityBone, out bindPose))
+            {
+                return bindPose;
+            }
+
             int index;
-            if (boneDict.TryGetValue (unityBone, out index)) {
-                bindPose = bindPoses [index];
-            } else {
-                bindPose = unityBone.worldToLocalMatrix * skinnedMesh.transform.localToWorldMatrix;
+            if (boneDict.TryGetValue(unityBone, out index))
```

Comment:
comment here (this is checking if it's a bone with a bind pose right?)

Reply:
yes